src/cli/cli.ts

```diff
@@ -3,13 +3,18 @@ import { hideBin } from 'yargs/helpers';
 import { ConstructAnalyzer } from '../lib/analyzer';
 
 /**
- * Converts snake_case signal names to Display Name format
- * Example: "weeklyDownloads" -> "Weekly Downloads"
+ * Converts signal names to Display Name format
+ * Examples:
+ * - "weeklyDownloads" -> "Weekly Downloads"
+ * - "numberOfContributors(Maintenance)" -> "Number Of Contributors (Maintenance)"
```

Comment:
but your PR description shows this:

```
Number Of Contributors ( Maintenance): 1
```

obviously if thats the case then this function is not working as intended